### DISCUSSION TITLE
react-script-editor -

### DISCUSSION THREAD

**Discussion by kierentoscan** [pnp/sp-dev-fx-webparts#3655] [Q&A] [answered]
### Disclaimer

Yes

### Sample

react-script-editor

### Contributor(s)

@wobba

### Question

Hi - I have a very newbie question here: We're currently using this script editor in a number of SharePoint pages in our company. Do we need to re-deploy this each time there is a version update?

Many thanks in advance!

**Comment by ghost** [comment]
Thank you for reporting this issue. We will be triaging your incoming issue as soon as possible.

**Comment by wobba** [comment, marked as answer]
No. As long as there is no breaking change in web part properties any SPFx web part which has a new version can be upgraded in-place.
